src/App.js

```diff
@@ -6,12 +6,60 @@ import {
   Redirect
 } from "react-router-dom"
 import './App.css';
+import About from "./About"
+import axios from "axios"
+import Dashboard from "./Dashboard"
+import Stock from "./Stock"
+import Search from "./Search"
+
 
 class App extends Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      stocks: [],
+      tracked: false
+    }
+    this.handleTrackedState = this.handleTrackedState.bind(this)
+  }
+  componentDidMount(){
+    axios.get("http://localhost3000/stocks").then((response)=> {
+      this.setState({
+        stocks: response.data
+      })
+    })
+  }
+  handleTrackedState(new){
+    let newStock=this.state.stocks
+    newStock.push(new)
```

Comment:
Keep in mind, `new` is a reserved keyword in javascript. This variable should be renamed to something like `incomingStock`